source/conversions.tex

```diff
@@ -280,7 +280,7 @@
 % NB: forbid line break between 'where' and 'cv'
 % to stop superscript j from running into
 % descender of p on the previous line.
-where~$cv_i^j$ denotes the cv-qualifiers in the cv-qualification signature of $\tcode{T}_j$:%
+where~$\cv{}_i^j$ denotes the cv-qualifiers in the cv-qualification signature of $\tcode{T}_j$:%
```

Comment:
A few lines below the same construct is expressed as ` $\text{\cv}_i^1$`. Seems like these should be consistent.

Reply:
Fixed.